com.unity.cinemachine/Editor/Upgrader/UpgradeObjectToCm3.cs

```diff
@@ -4,6 +4,8 @@
 using System;
 using System.Linq;
 using Cinemachine.Utility;
+using Codice.Client.BaseCommands;
+using Mono.Cecil;
```

Comment:
This is not needed. What's this?

Reply:
The new version of VisualStudio sometime adds `using` statements all by itself.  I haven't figured out what causes it, or how to make it stop.  It's very irritating.

Reply:
Thanks for catching it